### PR TITLE
chore: add support opt chain for EVM CCR

### PR DESCRIPTION
**Pull request by kamleshmugdiya**
Ticket: COIN-693

### CODE REVIEW

File: src/helpers/config.ts [hunk 1]
```diff
@@ -737,6 +737,14 @@ export const evmCCRIntendedChainCoins: Record<string, readonly CoinMetadata[]> =
       allCoinMetas.tavaxc,
       allCoinMetas.tetc,
     ] as const,
+    topeth: [
+      allCoinMetas.tarbeth,
+      allCoinMetas.hteth,
```
Comment (mullapudipruthvik) [inline src/helpers/config.ts:742]:
```suggestion
      allCoinMetas.tarbeth,
      allCoinMetas.hteth,
      allCoinMetas.tpolygon,
      allCoinMetas.tavaxc,
      allCoinMetas.tetc
```

File: src/helpers/config.ts [hunk 2]
```diff
@@ -737,6 +737,14 @@ export const evmCCRIntendedChainCoins: Record<string, readonly CoinMetadata[]> =
...
+    ] as const,
+    opeth: [
+      allCoinMetas.arbeth,
+      allCoinMetas.eth,
+    ] as const,
```
Comment (mullapudipruthvik) [inline src/helpers/config.ts:747]: same here

File: src/helpers/config.ts (hunk 1 above)
Comment (mullapudipruthvik) [reply src/helpers/config.ts:742]: why only eth and arb ?

**Review by mullapudipruthvik — CHANGES_REQUESTED** [review]

File: src/helpers/config.ts (hunk 2 above)
Comment (kamleshmugdiya) [reply src/helpers/config.ts:753]: added polygon, avaxc, etc

**Review by mullapudipruthvik — APPROVED** [review]

**Pull request merged**